application/tests/org.openjdk.jmc.rjmx.test/src/test/java/org/openjdk/jmc/rjmx/test/JVMSupportToolkitTest.java

```diff
@@ -46,7 +47,7 @@
 
 @SuppressWarnings("nls")
 public class JVMSupportToolkitTest {
-
+	private static final String ORACLE = "Oracle";
```

Comment:
Non-blocking, but we might want to name this a little more descriptively, like VENDOR_ORACLE?